app/rollup/index.py

```diff
@@ -0,0 +1,5 @@
+import algoliasearch_django as algoliasearch
+
+from .models import Rollup
+
+algoliasearch.register(Rollup)
```

Comment:
@jonseitz implemented three indices in Algolia: prod_SEC_DIRECTORY, dev_SEC_DIRECTORY, test_SEC_DIRECTORY. 

See the last code example in the docs for how to specify an index. Index name should come from an environment variable. https://www.algolia.com/doc/framework-integration/django/setup/?client=python

`fields` shouldn't be needed, but please include `settings = {'searchableAttributes': ['name']}` so that only the name field is searchable. We don't want people to get results based on office numbers or building.

Reply:
Ah I see, I originally thought we would allow searching for the `location` field as well. Thanks!